scripts/ci-reference-docs-build.sh

```diff
@@ -17,4 +24,4 @@ yarn packages:docsToMarkdown
 
 # cleaning packages
 echo "cleaning up packages build files..."
-lerna run clean
+yarn packages:clean
```

Comment:
I think this should be moved into the if block starting at line 8, since I don't think this cleanup should be part of CI (it will only slow things down and possibly break a parallel step).

Reply:
ok ok, sounds good! I will remove it.